tests/observability/metrics/conftest.py

```diff
@@ -916,3 +916,25 @@ def generated_api_deprecated_requests(prometheus):
     for _ in range(COUNT_FIVE):
         len(list(VirtualMachine.get()))
     return initial_metric_value + COUNT_FIVE
+
+
+@pytest.fixture()
+def smartclone_support_drivers():
+    return [
+        line.split()[1]
+        for line in run_command(command=shlex.split("oc get volumesnapshotclass"), check=False)[1]
+        .strip()
+        .split("\n")[1:]
+    ]
```

Comment:
We have a function `is_snapshot_supported_by_sc`, can you reuse it?

In general, if we need to get the resources, this is the way: `VolumeSnapshotClass.get()`

You can do either 
`list(VolumeSnapshotClass.get())` or `for vs in VolumeSnapshotClass.get()`